lua/completion/hover.lua

```diff
@@ -351,9 +351,11 @@ M.autoOpenHoverInPopup = function()
           M.winnr = winnr
         end
       else
+        local has_hover = false
         for _, value in pairs(vim.lsp.buf_get_clients(0)) do
-          if value.resolved_capabilities.hover == false then return end
+          has_hover = value.resolved_capabilities.hover or has_hover
```

Comment:
people are probably not going to have tons of clients, but perhaps we can still exit early here just in case?

```suggestion
          if value.resolved_capabilities.hover then
            has_hover = true
            break
          end
```

(I'm not a maintainer, take this as a light suggestion 😁)